navmodel-samples/src/main/kotlin/com/bumble/appyx/navmodel/tiles/operation/DeselectAll.kt

```diff
@@ -12,21 +14,10 @@ class DeselectAll<T : Any> : TilesOperation<T> {
 
     override fun invoke(
         elements: TilesElements<T>
-    ): NavElements<T, Tiles.TransitionState> =
-        elements.map {
-            if (it.targetState == Tiles.TransitionState.SELECTED) {
-                it.transitionTo(
-                    newTargetState = Tiles.TransitionState.STANDARD,
-                    operation = this
-                )
-            } else {
-                it
-            }
+    ): NavElements<T, Tiles.State> =
+        elements.transitionTo(STANDARD) {
+            it.targetState == SELECTED
         }
-
-    override fun equals(other: Any?): Boolean = this.javaClass == other?.javaClass
```

Comment:
This isn't a data class. Are you sure we should delete this?

Reply:
No, it was most probably accidental removal.